packages/web/src/store/account/sagas.js

```diff
@@ -69,6 +69,25 @@ import mobileSagas, { setHasSignedInOnMobile } from './mobileSagas'
 
 const NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
 
+const IP_STORAGE_KEY = 'user-ip-timestamp'
+
+function* recordIPIfNotRecent() {
+  const twentyFourHours = 24 * 60 * 60 * 1000
+  const now = Date.now()
+  const twentyFourHoursAgo = now - twentyFourHours
+  const storedIPStr = window.localStorage.getItem(IP_STORAGE_KEY)
+  const storedIP = storedIPStr && JSON.parse(storedIPStr)
+  if (!storedIP || storedIP.timestamp < twentyFourHoursAgo) {
```

Comment:
Should we include user_id / handle in this localstorage object, so that if a single user has multiple accounts on a machine we assign IPs for all of them?

Reply:
good idea